common/test/common_tests/types/text_test.cljc

```diff
@@ -0,0 +1,277 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns common-tests.types.text-test
+  (:require
+   [app.common.types.text :as cttx]
+   [clojure.test :as t :include-macros true]))
+
+
+(def content-base
```

Comment:
Is there any way to add test helpers to build this objects programatically, using the real code that generates text content? Having hard-coded values like this will prevent the tests to detect a possible future case in that the text chapes change their format and this breaks the sync algorithm.